api/lib/source/cosmosV0-source.js

```diff
@@ -238,19 +238,28 @@ class CosmosV0API extends RESTDataSource {
       '/staking/pool'
     )
     if (!Array.isArray(response)) return []
-    const proposals = response.map((proposal) => {
```

Comment:
why did this have to change?

Reply:
I don't know if I understand your question. It is better for performance to parallelize using Promise.all, so it is simply a performance improvement here